client/dump/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ IF(USING_LOCAL_BOOST)
     -DBOOST_SYSTEM_NO_DEPRECATED)
 ENDIF()
 
-INCLUDE_DIRECTORIES(SYSTEM ${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR})
+INCLUDE_DIRECTORIES(SYSTEM ${BOOST_PATCHES_DIR} ${BOOST_INCLUDE_DIR}/include)
```

Comment:
Pls, make the sure the precise location of `${BOOST_INCLUDE_DIR}`. `${BOOST_INCLUDE_DIR}`, is it point to `/path/to/boost/` or `/path/to/boost/include`?

Reply:
${BOOST_INCLUDE_DIR} is：`/usr/locak/stonedb_boost` on my instance.

Reply:
> Pls, make the sure the precise location of `${BOOST_INCLUDE_DIR}`. `${BOOST_INCLUDE_DIR}`, is it point to `/path/to/boost/` or `/path/to/boost/include`?

${BOOST_INCLUDE_DIR} is：/usr/locak/stonedb_boost on my instance.